UI/script.js

```diff
@@ -0,0 +1,47 @@
+document.addEventListener('DOMContentLoaded', function(){
+const list = document.querySelector('.product');
+const forms = document.forms;
+
+ // filter products
+ const searchBar = forms['search-product'].querySelector('input');
+ searchBar.addEventListener('keyup', (e) => {
+   const term = e.target.value.toLowerCase();
+   const products = list.getElementsByClassName('product');
+   Array.from(products).forEach((product) => {
+     const title = product.firstElementChild.textContent;
+     if(title.toLowerCase().indexOf(e.target.value) != -1){
+       book.style.display = 'block';
+     } else {
+       book.style.display = 'none';
+     }
+   });
+ });
+
+ function showAlert() {
+   alert("Your order is being processed, thank you");
+   
+ };
+
+//Declaring input information
+var name = document.getElementById("name").value;
+var location = document.getElementById("location").value;
+var phone = document.getElementById("phone-no").value;
+var email = document.getElementById("email").value;
+
+// storage of input information
+localStorage.setItem("Name",name);
+localStorage.setItem("Location",location);
+localStorage.setItem("Phone",phone);
+localStorage.setItem("Email",email);
```

Comment:
@wamani-will  these 4 lines of code can also be written like
``` 
let personDetails  = {
"Name": name,
"Location": location,
"Phone": phone,
"Email": email
}
localStorage.setItem("pdetails", JSON.stringfy(productDetails))
```

The reason why l use `JSON.stringfy` is to convert the object into a JSON object.

Reply:
Thanks, I saw the JSON idea but I thought it was out of vanilla JavaScript